client/components/search/index.jsx

```diff
@@ -38,7 +38,8 @@ module.exports = React.createClass( {
 		onKeyDown: React.PropTypes.func,
 		disableAutocorrect: React.PropTypes.bool,
 		onBlur: React.PropTypes.func,
-		searching: React.PropTypes.bool
+		searching: React.PropTypes.bool,
+		forceCloseButton: React.PropTypes.bool
```

Comment:
> Maybe we're doing something funky with <Search>?

Nope, nothing funky going on :)

@mtias makes a fair point though - do we want to show the X when the input is empty?